### PR TITLE
Feature/comment display

### PR DESCRIPTION
**Pull request by colefiscus**
#### What does  this PR do?

Adds functionality that automatically shows added comments to comment modal. Also allows user to delete comments.

When deleting a comment, the comment is removed from the modal, the comment ticker is updated.

Adds styling for comments when added to modal.

#### Where should the reviewer start?

Reviewer should look at lines 180-190 and 226-251 to ensure functionality for comments are bug-free and concise.

#### How should this be manually tested?

Tester should add comments, delete comments, and refresh page to make sure that comments are removed from local storage and savedComments array, while also making sure functions work as expected.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,6 +32,9 @@ <h2>Filter Starred Ideas</h2>
         <textarea class="comment-text" rows=4 type="text" name="comment-text" placeholder="Leave a Comment!"></textarea>
         <button type="submit" class="submit-comment" disabled>Add Comment</button>
       </div>
+      <h1 class="comment-header">Comments</h1>
+      <div class="comment-display">
```
Comment (DrewBradley) [inline index.html:36]: I like the way we integrated the comments to be displayed in the same 'modal' window as the comment submission form. So neat!

File: styles.css [hunk 2]
```diff
@@ -221,6 +221,34 @@ p {
   animation-duration: 0.4s;
 }
 
+.comment-display {
```
Comment (JeffKersting) [inline styles.css:224]: Fix height of idea input area so inputs do not overflow when page size is reduced.

File: src/main.js [hunk 3]
```diff
@@ -89,12 +90,11 @@ function checkEventTarget(event) {
     favoriteIdea(eventTarget);
   } else if (eventTarget.classList.contains("comment-button")) {
     var id = eventTarget.parentNode.parentNode.id;
-    showCommentInput();
+    showCommentInput(id);
```
Comment (DrewBradley) [inline src/main.js:93]: Utilizing a parameter to allow passing the id into a new function. Well done!

File: src/main.js [hunk 4]
```diff
@@ -178,8 +177,17 @@ function searchIdeas() {
...
 
-function showCommentInput() {
+function showCommentInput(id) {
+  commentDisplay.innerHTML = '';
   modal.style.display = 'block';
+  var newId = parseInt(id)
+  for (var i = 0; i < savedComments.length; i++) {
+    if (savedComments[i].parentId === newId) {
+      commentDisplay.innerHTML += `
+        <span id=${savedComments[i].id} class="comment-delete">&#9746</span><p class="comment-display-text">${savedComments[i].text}</p>
```
Comment (JeffKersting) [inline src/main.js:187]: Could try to break this into more lines to make element creation clearer.

File: src/main.js [hunk 5]
```diff
@@ -227,3 +222,30 @@ function addCommentToIdea(newComment, parentId) {
...
+        savedComments[i].deleteFromStorage()
+        savedComments.splice(i, 1)
+        showIdeas(savedIdeas.sort(compareId));
+      }
+    }
+    modal.style.display = 'none'
+  }
+}
+
+function checkIdeasArray(id, targetId) {
```
Comment (DrewBradley) [inline src/main.js:241]: This is undoubtedly the most prolific piece of code written during this project. I am proud of how we tackled the challenge of only deleting the individual comment from the idea card it is associated with. We should all be very proud.

**Review by DrewBradley — APPROVED** [review]

**Review by JeffKersting — APPROVED** [review]

**Pull request merged**